.evergreen/install-dependencies.sh

```diff
@@ -1,79 +1,97 @@
 #!/usr/bin/env bash
-
-set -o xtrace   # Write all commands first to stderr
 set -o errexit  # Exit the script with error if any of the commands fail
 
-NODE_BINDINGS_PATH="${PROJECT_DIRECTORY}/bindings/node"
-NODE_ARTIFACTS_PATH="${NODE_BINDINGS_PATH}/node-artifacts"
-NPM_CACHE_DIR="${NODE_ARTIFACTS_PATH}/npm"
-NPM_TMP_DIR="${NODE_ARTIFACTS_PATH}/tmp"
-BIN_DIR="$(pwd)/bin"
-
-NVM_WINDOWS_URL="https://github.com/coreybutler/nvm-windows/releases/download/1.1.9/nvm-noinstall.zip"
-NVM_URL="https://raw.githubusercontent.com/creationix/nvm/v0.38.0/install.sh"
-
-# this needs to be explicitly exported for the nvm install below
-export NVM_DIR="${NODE_ARTIFACTS_PATH}/nvm"
-
-# create node artifacts path if needed
-mkdir -p "${NODE_ARTIFACTS_PATH}"
-mkdir -p "${NPM_CACHE_DIR}"
-mkdir -p "${NPM_TMP_DIR}"
-mkdir -p "${BIN_DIR}"
+NODE_LTS_NAME=${NODE_LTS_NAME:-fermium}
+NODE_ARTIFACTS_PATH="${PROJECT_DIRECTORY:-$(pwd)}/node-artifacts"
+if [[ "$OS" = "Windows_NT" ]]; then NODE_ARTIFACTS_PATH=$(cygpath --unix "$NODE_ARTIFACTS_PATH"); fi
+
+mkdir -p "$NODE_ARTIFACTS_PATH/npm_global"
+
+# Comparisons are all case insensitive
+shopt -s nocasematch
+
+# index.tab is a sorted tab separated values file with the following headers
+# 0       1    2     3   4  5  6    7       8       9   10
+# version date files npm v8 uv zlib openssl modules lts security
+curl --retry 8 -sS "https://nodejs.org/dist/index.tab" --max-time 300 --output node_index.tab
+
+while IFS=$'\t' read -r -a row; do
+  node_index_version="${row[0]}"
+  node_index_date="${row[1]}"
+  node_index_lts="${row[9]}"
+  [[ "$node_index_version" = "version" ]] && continue # skip tsv header
+  [[ "$NODE_LTS_NAME" = "latest" ]] && break # first line is latest
+  [[ "$NODE_LTS_NAME" = "$node_index_lts" ]] && break # case insensitive compare
+done < node_index.tab
+
+if [[ "$OS" = "Windows_NT" ]]; then
+  operating_system="win"
+elif [[ $(uname) = "darwin" ]]; then
+  operating_system="darwin"
+elif [[ $(uname) = "linux" ]]; then
+  operating_system="linux"
+else
+  echo "Unable to determine operating system: $operating_system"
+  exit 1
+fi
 
-export NVM_DIR="${NODE_ARTIFACTS_PATH}/nvm"
-mkdir -p "${NVM_DIR}"
+architecture=$(uname -m)
+if [[ $architecture = "x86_64" ]]; then
+  architecture="x64"
+elif [[ $architecture = "arm64" ]]; then
+  architecture="arm64"
+elif [[ $architecture = "aarch64" ]]; then
+  architecture="arm64"
```

Comment:
This is the only change from the original script, I'll open a PR to update the script on 5.x so it is sync-ed